src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -98,6 +118,13 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example for above configuration:
+ * </p>
+ * <pre>
+ * public void test1(){ } //OK
```

Comment:
https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples please read "ATTENTION" paragraph, java code example should be the same for all configs , only trailing comments are changing.

example https://github.com/checkstyle/checkstyle/commit/418fdb1a691a4241eb93d1282637b16f184d58e6